best-practices/database/README.md

```diff
@@ -1,4 +1,9 @@
-# DATABASE
+# Databases
+A database is an organized collection of data, generally stored and accessed electronically from a computer system. Where databases are more complex they are often developed using formal design and modeling techniques. 
+
+At Bitwise Industries, we generally use T-SQL and NoSQL databases in our projects. More information can be found [here](https://www.altexsoft.com/blog/business/comparing-database-management-systems-mysql-postgresql-mssql-server-mongodb-elasticsearch-and-others/).
```

Comment:
I didn't realized we had projects that used T-SQL

Reply:
I kinda used this as a blanket term....but now that you brought it up, I think this is incorrect.

T-SQL is just something I got used to saying at an old job, but it really should be SQL.

Reply:
fixed via: 94db7296ec241a09378c4746d3ce99efcf8e304a